portablemc/cli/lang.py

```diff
@@ -168,6 +172,7 @@ def get(key: str, **kwargs) -> str:
     "error.socket": "This operation requires an operational network, but a socket error happened:",
     "error.socket.tip.version_manifest": "Version manifest may not be locally cached, try to run this command once with an operational network.",
     "error.socket.tip.fabric_loader_version": "Fabric loader version must be specified if network is not operational.",
+    "error.socket.tip.legacyfabric_loader_version": "Fabric loader version must be specified if network is not operational.",
```

Comment:
Typo: LegacyFabric instead of Fabric.

Reply:
LegacyFabric uses regular fabric-loader. Should I just not add the extra lang for it?